dash/dash.py

```diff
@@ -754,15 +755,47 @@ def _validate_callback(self, output, inputs, state, events):
             ).replace('    ', ''))
 
         callback_id = _create_callback_id(output)
-        if callback_id in self.callback_map:
-            raise exceptions.CantHaveMultipleOutputs('''
+        is_multi = isinstance(output, (list, tuple))
+        callbacks = set(itertools.chain(*(
+            x[1:-1].split(':')
+            if x.startswith('[')
+            else [x]
+            for x in self.callback_map
+        )))
+        ns = {
+            'duplicates': callback_id
```

Comment:
Why is this initialized to `callback_id`? (It does not look to be used below, same result if it was initialized to `'duplicates': None`)